project_event/models/project_task.py

```diff
@@ -727,6 +727,50 @@ def get_booked_resources(self):
                             '<br>'
         return res
 
+    def get_double_booked_resources(self, date_start=None, date_end=None):
```

Comment:
hey, this methods are duplicated code. Just a comment, but you could reference the object and without creating it and make these verifications.